ReadMe.md

```diff
@@ -52,13 +52,6 @@
 - **Use professional documentation**
 - **Use HTML and CSS**
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- [Live Demo Link](https://google.com)
```

Comment:
- [ ] I will suggest that you include this section and just put `N/A` instead of including a false link. :+1: eg.

```
## 🚀 Live Demo <a name="live-demo"></a>

- N/A
```